go/client/chat_resolver.go

```diff
@@ -229,7 +226,11 @@ func (r *chatConversationResolver) Resolve(ctx context.Context, req chatConversa
 					info.Visibility, info.Triple.TopicType, info.TopicName, info.TLFNameExpandedSummary())
 			}
 		}
-		return &conversation, false, nil
+		var err error
+		if conversation.Error != nil {
```

Comment:
Why bother making this an error at all?

Reply:
I was leaving as is in case other callers depended on it but  i guess we can just remove it